schemas/data/first-quartiles.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/advertising/first-quartiles",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "first-quartiles",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/metric"],
+  "description": "The advertising metric first-quartiles describes digital video ads which played continuously for at least 25% of the total duration at normal speed",
```

Comment:
"played continuously"  seems quite specific. Does it mean no pauses, either human or buffering for example, or does it mean the first 25% was watched "somehow"?

Reply:
Cleaned up description.